platform-sdk/docs/core/dynamic-address-book/testing/GreenWire-Test-Plan.md

```diff
@@ -0,0 +1,116 @@
+# Green Wire - Event Birth Round as Ancient Threshold
+
+Testing of the Green Wire is broken up into the following phases:
+
+* Component Unit Tests
+* Pipeline Unit Tests
+* JRS Tests
+
+## Implementation
+
+* Introduce `AncientMode` enumeration to switch between `GENERATION_THRESHOLD` and `BIRTH_ROUND_THRESHOLD`.
+* Introduce `AncientThreshold` that changes definition based on `AncientMode`.
+    * when in `GENERATION_THRESHOLD` mode, `AncientThreshold` is the minimum generation of the judges of
+      round (`pendingConsensusRound` - `roundsNonAncient`).
+    * when in `BIRTH_ROUND_THRESHOLD` mode, `AncientThreshold` is the minimum birth round of the judges of
+      round (`pendingConsensusRound` - `roundsNonAncient`).
+* Introduce `NonAncientEventWindow` (aka `EventWindow`) that contains `AncientThreshold` for both `GENERATION_THRESHOLD`
+  and `BIRTH_ROUND_THRESHOLD`.
+* Events implement `AncientIndicator(AncientMode)` which returns their generation in `GENERATION_THRESHOLD` mode and
+  birth round in `BIRTH_ROUND_THRESHOLD` mode.
+* Events are ancient when `Event.getAncientIndicator(AncientMode) < EventWindow.getAncientThreshold(AncientMode)`.
+* Consensus:
+    * Rename `MinGenInfo` to `MinJudgeInfo` which contains the minimum generation or birth round of judges depending on
```

Comment:
It was actually renamed to `MinimumJudgeInfo`

Reply:
correcting it